ludwig/hyperopt/execution.py

```diff
@@ -145,6 +145,9 @@ def __init__(
         scheduler: Optional[Dict] = None,
         **kwargs,
     ) -> None:
+        # Force-populate the search algorithm registry
+        import ludwig.hyperopt.search_algos  # noqa
```

Comment:
@justinxzhao Not sure who to ping but I think you'll have a good answer. I have to import this module somewhere before search algorithm initialization so that the registry is not empty. 

In my original giant PR I had put this inside the declaration for `BaseSearchAlgorithmConfig` (see: https://github.com/ludwig-ai/ludwig/blob/26cda9d8d2a8ec747a012fe4efa92a447a39f632/ludwig/schema/hyperopt/search_algorithm.py#L13). But when breaking up that PR into parts, in #2781 I had to import it inside of the `init` for `RayTuneExecutor` (as seen here). Is there one place or another that is better for this?

Reply:
Does `ludwig/hyperopt/__init__.py` work?

Reply:
yup! the simplest of places in hindsight...